compiler/src/dotty/tools/dotc/transform/PickleQuotes.scala

```diff
@@ -95,9 +95,8 @@ class PickleQuotes extends MacroTransform {
         case Apply(Select(quote: Quote, nme.apply), List(quotes)) =>
           val (contents, quote1) = makeHoles(quote)
           val quote2 = encodeTypeArgs(quote1)
-          val contents1 = contents ::: quote.tags
-          val pickled = PickleQuotes.pickle(quote2, quotes, contents1)
-          transform(pickled) // pickle quotes that are in the contents
+          val contents1 = contents.map(transform(_)) ::: quote.tags
```

Comment:
By the way, why are the tags after the content? Aren't they referenced in the body?

Reply:
The tags will not contain quotes, they are all `TermRef`s. Therfore we know that this transformation will have no effect.

Reply:
right, I mean, why isn't it `quote.tags ::: contents.map(...)`

Reply:
The order was irrelevant as we split them in `PickleQuotes.pickle`. This also deserves some optimization. I will add a commit for that.

Reply:
It was added at the end because of a zipWithIndex that we needed for the contents but not the tags.